frontend/app/screens/Sessions/index.js

```diff
@@ -0,0 +1,200 @@
+// @flow
+
+import React from 'react';
+import Helmet from 'react-helmet';
+import Header from 'Header';
```

Comment:
Don't bother bringing in any page components or styles in the future since all that is probably gonna change when designs are finished